virtcontainers/utils/utils.go

```diff
@@ -275,3 +276,23 @@ const (
 	MiB          = KiB << 10
 	GiB          = MiB << 10
 )
+
+// Binary to use to log program output
+const LoggerBinryName = "systemd-cat"
```

Comment:
It will print the output in the format of cloud hypervisor stdout, I am afraid is not structured logged. 
```
journalctl -t kata-hypervisor -f
Apr 28 21:11:15 jcvenega kata-hypervisor[102976]: cloud-hypervisor: 10.342620707s: DEBUG:vm-virtio/src/vsock/device.rs:161 -- vsock: epoll_handler::process_tx()
Apr 28 21:11:15 jcvenega kata-hypervisor[102976]: cloud-hypervisor: 10.342638908s: DEBUG:vm-virtio/src/vsock/device.rs:119 -- vsock: epoll_handler::process_rx()
Apr 28 21:11:15 jcvenega kata-hypervisor[102976]: cloud-hypervisor: 10.34267121s: DEBUG:vm-virtio/src/vsock/unix/muxer.rs:178 -- vsock muxer: RX pkt: [2, 0, 0, 0, 0, 0, 0, 0, 3, 0, 0, 0, 0, 0, 0, 0, 51, 4, 0, 64, 0, 4, 0, 0, 17, 0, 0, 0, 1, 0, 5, 0, 0, 0, 0, 0, 0, 0, 1, 0, 231, 1, 0, 0]
Apr 28 21:11:15 jcvenega kata-hypervisor[102976]: cloud-hypervisor: 10.342782016s: DEBUG:vm-virtio/src/vsock/device.rs:105 -- vsock: raising IRQ
```

Reply:
@jodh-intel @sboeuf I wonder if it worth I take a look to structured log from kata-agent in rust( I think now it supports it, rigth?) and I try to use it in cloud-hypervisor

Reply:
Yes, the new rust agent already uses structured logging. If clh wants to, they could just grab:

- https://github.com/kata-containers/kata-containers/tree/master/src/agent/logging
  (see also https://github.com/kata-containers/kata-containers/issues/46).